pkg/guid/guid_pool.go

```diff
@@ -35,45 +33,37 @@ type GuidPool interface {
 
 type guidPool struct {
 	kubeClient        k8s_client.Client // kubernetes client
-	rangeStart        net.HardwareAddr  // first guid in range
-	rangeEnd          net.HardwareAddr  // last guid in range
-	currentGUID       net.HardwareAddr  // last given guid
-	guidPoolMap       map[string]bool   // allocated guid map and status
+	rangeStart        GUID              // first guid in range
+	rangeEnd          GUID              // last guid in range
+	currentGUID       GUID              // last given guid
+	guidPoolMap       map[uint64]bool   // allocated guid map and status
 	guidPodNetworkMap map[string]string // allocated guid mapped to the pod and network
```

Comment:
so this guidPodNetworkMap  will be remove in the other PR right?

Reply:
It should be in #54